data/human/news.txt

```diff
@@ -801,6 +871,26 @@ news "geological engineer"
 
 
 
+news "electrical engineer"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	name
+		word
+			"Electrical engineer"
+			"Power networks expert"
+	message
+		word
+			`"`
+		word
+			"Earth has a weird power-distribution network, where the polarity of home outlets changes 50 or 60 times per second. Every other planet just uses a positive and negative polarity for outlets."
+			"Earth has a beautiful power-distribution network, where the polarity of home outlets changes 50 or 60 times per second. Other planets use DC current and need to add measures to ensure that polarity is matched, but on earth the outlets are symmetric by default."
+			"Voltage conversion from DC power to DC power works extremely well on every planet except Earth. Earth's ancient power-network still uses AC power for distribution."
+			"Most equipment works from 100 Volt to 500 Volt power, but just check compatibility before connecting your equipment."
```

Comment:
Is Volt supposed to be capitalized?

Reply:
Probably not. I changed it to lower-case.